roles/container-engine/cri-o/vars/fedora.yml

```diff
@@ -4,3 +4,9 @@ crio_packages:
   - cri-tools
 
 crio_conmon: /usr/libexec/crio/conmon
+
+crio_kubernetes_version_matrix:
+  "1.18": "1.18"
+  "1.17": "1.17"
+
+crio_version: "{{ crio_kubernetes_version_matrix[crio_required_version] | default('1.17') }}"
```

Comment:
It would be better to add `# NOTE: Need to specify 1.17 for fedora instead of the default 1.19 due to test failure` or something because it is a little tricky to specify different version here from the default one.

Reply:
Ok will do